src/JsonSchema/Uri/UriRetriever.php

```diff
@@ -40,19 +40,19 @@ class UriRetriever implements BaseUriRetrieverInterface
      *
      * @param UriRetrieverInterface $uriRetriever
      * @param string $uri
-     * @return bool|void
+     * @return bool
      */
     public function confirmMediaType($uriRetriever, $uri)
     {
         $contentType = $uriRetriever->getContentType();
 
         if (is_null($contentType)) {
             // Well, we didn't get an invalid one
-            return;
+            return true;
```

Comment:
Given that (in master) the function could return `true` when it's a schema from json-schema.org, and `null` if it is explicitly valid, it just isn't seem all that logical.  The only usage internally just relies on it halting with an exception if it's invalid.

Is it even feasible that someone has written code to handle the different return types?  If you're using the library for validation, it's going to be called again anyway and you won't have any control of the result..

That said, maybe there is an argument for leaving it as it is, because the return value is basically irrelevant anyway!